plugin/git-slides.vim

```diff
@@ -1,14 +1,32 @@
-:set nofoldenable
-
-:nmap <Space> :wa<CR>:call system("git-slides next")<CR>:e!<CR>
-:nmap <Backspace> :wa<CR>:call system("git-slides prev")<CR>:e!<CR>
-:nmap sl :wa<CR>:call system("git-slides next")<CR>:e!<CR>
-:nmap sh :wa<CR>:call system("git-slides prev")<CR>:e!<CR>
-:nmap sL :call system("git-slides next -f")<CR>:e!<CR>
-:nmap sH :call system("git-slides prev -f")<CR>:e!<CR>
-:nmap ss :wa<CR>:call system("git-slides save")<CR>:e!<CR>
-:nmap sb :wa<CR>:call system("git-slides insert-before")<CR>:e!<CR>
-:nmap sa :wa<CR>:call system("git-slides insert-after")<CR>:e!<CR>
-:nmap st :wa<CR>:call system("git-slides transform")<CR>:e!<CR>
-:nmap sd :wa<CR>:call system("git-slides delete")<CR>:e!<CR>
-:nmap sr :!./%<CR>
+" Vim global plugin for interfacing with git-slides
+
+if !get(g:, 'gitslides_use_custom_mappings')
+  let g:gitslides_use_custom_mappings = 0
+endif
+
+nnoremap <silent> <Plug>GitSlidesNext :wa<CR>:call system("git-slides next")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesPrevious :wa<CR>:call system("git-slides prev")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesNextForce :call system("git-slides next -f")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSildesPreviousForce :call system("git-slides prev -f")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesSave :wa<CR>:call system("git-slides save")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesInsertBefore :wa<CR>:call system("git-slides insert-before")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesInsertAfter :wa<CR>:call system("git-slides insert-after")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesTransform :wa<CR>:call system("git-slides transform")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesDelete :wa<CR>:call system("git-slides delete")<CR>:e!<CR>
+nnoremap <silent> <Plug>GitSlidesExecute :!./%<CR>
+
+if !g:gitslides_use_custom_mappings
+  nmap <Space> <Plug>GitSlidesNext
```

Comment:
When I use your version and press `<Space>`, it now takes about a second to move to the next slide, whereas with my version it was instant, and `<Backspace>` is still instant. Does that happen on your machine too? Any idea what might be causing this?

Reply:
I don't think so, but I'll check again.

Reply:
Ok, after checking again (this time after some sleep) it does happen on my machine aswell, `<Backspace>` is instant as well which only makes this stranger.

I read the documentation regarding `<Plug>` carefully (it isn't much) and I can't see any reason for this behaviour.

I'll investigate it a bit further, if I can't figure it out I'll change the PR to keep the mappings as they were but allow the possibility of now loading the default mappings (I for example already have several things mapped to some of these mappings).

Is that acceptable?

Reply:
Sure

Reply:
I removed the <Plug> as I couldn't crack it.. : (

I did however leave the mappings as 'nnoremap' to avoid recursion.